src/dotnet/APIView/ClientSPA/src/app/_components/review-page/review-page.component.html

```diff
@@ -9,7 +9,14 @@
         (pageOptionsEmitter)="handlePageOptionsEmitter($event)"></app-review-info>
     <div class="mt-2" style="display: flex;">
         <div>
-            <p-menu [model]="sideMenu"></p-menu>
+            <p-menu [model]="sideMenu" class="side-menu">
```

Comment:
what is this adding?

Reply:
![side Menu](https://github.com/user-attachments/assets/1637fcef-2215-4e43-949e-e9cc96d68f0f)